flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/TableSourceITCase.scala

```diff
@@ -329,4 +331,44 @@ class TableSourceITCase extends BatchTestBase {
       )
     )
   }
+
+  @Test
+  def testProjectablePoJoTable(): Unit = {
```

Comment:
What's the purpose of this test? I revert the changes in `TableSourceUtil` and this test is still passed.

Reply:
I think we should have a test that there is a DECIMAL type defined the source which may should use DDL.